### PR TITLE
Add desktop version

### PR DESCRIPTION
**Pull request by Lusindiso**
## Here I have made these changes:
- Add navigation for desktop version.
- Add desktop version of the header section.
- Add desktop version and projects for desktop version, and adjusted mobile styles.
- Add desktop version for about section
- add a different form for desktop and kept the old for mobile, to avoid validation errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,259 +14,315 @@
...
+  </div>
+</section>
+<footer class="contact-form">
+  <form action="https://formspree.io/f/mgedzjjn" method="POST" class="form desktop">
+    <h3 class="form__heading">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+    <ul class="form__list form__list--big">
+        <li class="form__list--item fullname">
+          <input type="text" name="first-name" maxlength="30" placeholder="First name" required class="first-name">
+
+          <input type="text" name="last-name" maxlength="30" placeholder="Last name" required class="last-name">
```
Comment (Amrhub) [inline index.html]: - [ ] I know it's a bit confusing but as per requirements let's make sure we only have one input which is `full name`
image

File: style.css [hunk 2]
```diff
@@ -177,26 +265,156 @@ a:link {
...
+    background-image: url('./img/project-4.jpg');
+  }
+
+  .works-project-5 {
+    background-image: url('./img/project-5.jpg');
+  }
+
+  .works-project-6 {
+    background-image: url('./img/project-6.jpg');
+  }
```
Comment (Amrhub) [inline style.css]: Great job implementing the UI design but there is a single issue, I know it was not so clear. 
- [ ] This portfolio template project is supposed to look like your last 5 projects and ONLY when hovered the description, title, and technologies are disappeared and the button is displayed (like your first example)

**Review by Amrhub — CHANGES_REQUESTED** [review]
Hi @Lusindiso,  👋🏻 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!
These changes aim to add some awesomeness to your awesome project and code skills.

## To Highlight ✔️ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Lusindiso) [reply index.html]: Thank you, I thought I had to implement this, I spend almost an hour trying to get this to work thank you.

File: style.css (hunk 2 above)
Comment (Lusindiso) [reply style.css]: I thought we weren't suppose to implement animation and transitions yet.

Comment (Amrhub) [reply style.css]: you could do it without animation or transition and when you get to this part just make it look fancier 🌞

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Lusindiso, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**